### PR TITLE
Split search

### PR DESCRIPTION
**Pull request by tkosciol**
Renamed pdb_search to split_search and added Click interface.

### CODE REVIEW

File: microprot/scripts/split_search.py [hunk 1]
```diff
@@ -550,3 +551,54 @@ def mask_sequence(hhsuite_fp, fullsequence_fp, subsequences_fp=None,
                 for type_ in results}
     except IOError:
         raise IOError('Cannot write to file "%s"' % subsequences_fp)
+
+
+def pretty_output(mask_out):
+    for key in mask_out.keys():
+        print(key)
+        for i in range(len(mask_out[key])):
+            print('\t>', mask_out[key][i][0], '\n\t', mask_out[key][i][1])
```
Comment (sjanssen2) [inline microprot/scripts/split_search.py]: the more pythonic way would be:
print('\t>%s\n\t%s' % (mask_out[key][i][0], mask_out[key][i][1])

**Comment by tkosciol** [comment]
I definitely don't want those `.nfs*` files. No idea how hidden files hot pushed.

**Comment by tkosciol** [comment]
@sjanssen2 please merge, if you think it's fine.

**Comment by sjanssen2** [comment]
mainly renaming, thus one review should be fine

**Pull request merged**